### PR TITLE
MF homepage

### PR DESCRIPTION
**Pull request by fritzschoff**

### CODE REVIEW

File: apps/backend-functions/src/orgs.ts [hunk 1]
```diff
@@ -184,6 +184,10 @@ export async function onOrganizationUpdate(change: Change<FirebaseFirestore.Docu
     await Promise.all(promises)
   }
 
+  if (await hasAcceptedMovies(after)) {
+    after['hasAcceptedMovies'] = true;
```
Comment (pldespaigne) [inline apps/backend-functions/src/orgs.ts:188]: why not `after.hasAcceptedMovies = true;`

File: apps/financiers/financiers/src/app/landing/landing.component.html [hunk 2]
```diff
@@ -36,7 +36,7 @@ <h5>FEATURES</h5>
           <mat-card *ngFor="let feature of sellerFeatures">
             <img [asset]="feature.imgAsset" type="logo" alt="seller feature image">
             <h3>{{ feature.title }}</h3>
-            <p class="mat-body-1">{{ feature.description }}</p>
+            <p class="mat-body-1" [innerHTML]="feature.description"></p>
```
Comment (pldespaigne) [inline apps/financiers/financiers/src/app/landing/landing.component.html:39]: That's seems reaaaaaaaally strange ^^
Maybe add an optional `link` property to the `feature` object instead

File: libs/utils/src/lib/algolia/algolia.interfaces.ts [hunk 3]
```diff
@@ -80,8 +81,15 @@ export type AlgoliaIndex = keyof typeof algoliaIndex;
 
 ///// TYPES //////
 
-interface AlgoliaRecord {
+export interface AlgoliaRecord<T> {
     objectID: string,
+    nbHits: number,
+    nbPages: number,
+    hits: T,
```
Comment (pldespaigne) [inline libs/utils/src/lib/algolia/algolia.interfaces.ts:88]: should it not be `T[]` ?

**Review by pldespaigne — CHANGES_REQUESTED** [review]

File: apps/backend-functions/src/orgs.ts (hunk 1 above)
Comment (fritzschoff) [reply apps/backend-functions/src/orgs.ts:188]: cause this is a algolia flag and doesn't exist on the OrganizationDocument

File: apps/financiers/financiers/src/app/landing/landing.component.html (hunk 2 above)
Comment (fritzschoff) [reply apps/financiers/financiers/src/app/landing/landing.component.html:39]: yes. that is a better idea

File: apps/financiers/financiers/src/app/landing/landing.component.html [hunk 4]
```diff
@@ -36,7 +36,13 @@ <h5>FEATURES</h5>
           <mat-card *ngFor="let feature of sellerFeatures">
             <img [asset]="feature.imgAsset" type="logo" alt="seller feature image">
             <h3>{{ feature.title }}</h3>
-            <p class="mat-body-1">{{ feature.description }}</p>
+            <p class="mat-body-1 features">{{ feature.description }}
+              <a *ngIf="feature?.link?.href" [href]="feature.link.href" target="_blank">
+                <u>
+                  {{ feature.link.text }}
+                </u>
```
Comment (pldespaigne) [inline apps/financiers/financiers/src/app/landing/landing.component.html:43]: why using `<ul>` ?

File: apps/financiers/financiers/src/app/marketplace/home/home.component.html [hunk 5]
```diff
@@ -55,10 +55,26 @@ <h1>Our Partners</h1>
...
+    </div>
+    <bf-carousel flex [min]="2">
+      <mat-card *ngFor="let benefit of benefits" carouselItem [col]="6">
+        <img [asset]="benefit.imgAsset" type="logo" alt="benefit feature image">
+        <h3>{{ benefit.title }}</h3>
+        <p class="mat-body-1 benefits">{{ benefit.description }}</p>
+        <a *ngIf="benefit?.link?.href" [href]="benefit.link.href" target="_blank">
+          <u>
+            {{ benefit.link.text }}
+          </u>
```
Comment (pldespaigne) [inline apps/financiers/financiers/src/app/marketplace/home/home.component.html:75]: why using `<ul>` ?

**Review by pldespaigne — APPROVED** [review]

File: apps/financiers/financiers/src/app/landing/landing.component.html (hunk 4 above)
Comment (fritzschoff) [reply apps/financiers/financiers/src/app/landing/landing.component.html:43]: u is for underline. its not a list.

**Pull request merged**